.github/workflows/ci.yml

```
@@ -53,6 +53,68 @@ jobs:
+  # binaries target macOS 12, so the run needs a Security.framework symbol
+  # stub and still hits Darling runtime bugs (sigtramp crashes under
+  # subprocess load, os/exec and TLS hangs) — last full run: 67/74 test
+  # packages passed, spectra CLI worked. When Darling can present macOS 12,
+  # delete the `if: false` below to re-enable; everything else is wired.
+  #
+  # The pinned release debs target Ubuntu 24.04 (noble) amd64; keep the
+  # runner pinned to match. Darling is userspace-only (darlingserver), so
+  # no kernel module is needed on hosted runners. zig cross-links the
+  # Security stub dylib.
+  darling-smoke:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)